engine/schema/src/main/java/com/cloud/upgrade/OfficialSystemVMTemplate.java

```diff
@@ -0,0 +1,86 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.cloud.upgrade;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.cloud.hypervisor.Hypervisor;
+
+public class OfficialSystemVMTemplate {
```

Comment:
The whole class is using 4.11.3 instead of using a map between CloudStack version and the corresponding system VM template version

Reply:
This code was added in an upgrade script, I moved it to a class.